### DISCUSSION TITLE
Running multiple climate (rainfall) predictions

### DISCUSSION THREAD

**Discussion by IsaDio** [pastas/pastas#586] [Q&A] [answered]
Hi,

I have a calibrated model I would like to use for predictions using multiple time series of rainfall (to simulate climate variability and a range of potential groundwater levels).
I have my rainfall time series setup so the warmup period and calibration period have the same rainfall data, but not the prediction period.
I thought this would be straightforward but I can't think of a way to modify the rainfall time series without generating a new recharge model and recalibrating (or setting the parameters). 
Is there a simple way to change the rainfall time series for a simulation?

Thank you for your help!!

**Comment by raoulcollenteur** [comment, marked as answer]
Hi @IsaDio,

For now this is a bit of a hack, but not that difficult. For a stressmodel named "rain", manually replace the series_original stored in the stressmodel object: 

`ml.stressmodels["rain"].stress[0].series_original = new_time_series
`
Over the next couple of month I will actually work on some way to run different scenarios more natively, but that will take some time.

Let me know if this works!

Cheers,
Raoul

**Reply by IsaDio** [reply]
Thank you @raoulcollenteur. 

It works with ml.stressmodels["recharge"].stress[0].series_original = new_time_series 

👍
